dxr/config.py

```diff
@@ -153,6 +154,7 @@ def __init__(self, input, relative_to=None):
 
         if not relative_to:
             relative_to = getcwd()
+        self.path = relative_to
```

Comment:
So we're assuming only 1 DXR config file per dir, then, since we don't include the file name (since we don't have access to it)?